src/splitWhen.js

```diff
@@ -0,0 +1,23 @@
+var _curry2 = require('./internal/_curry2');
+var complement = require('./complement');
+var dropWhile = require('./dropWhile');
+var takeWhile = require('./takeWhile');
+
+
+/**
+ * Splits a given list at the first occurance of a satisfied predicate.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean)t  -> [[a], [a]]
+ * @param {Function} fn The predicate that determines where the array is split.
+ * @param {Array} list The array to be split.
+ * @return {Array}
+ * @example
+ *
+ *      R.splitWhen(R.equals(2), [1, 2, 3]);   //=> [[1], [2, 3]]
```

Comment:
Let's use `[1, 2, 3, 1, 2, 3]` as the input list, to demonstrate that splitting occurs in one place only.